src/rdf4cpp/Literal.cpp

```diff
@@ -409,12 +409,12 @@ Literal Literal::try_get_in_node_storage(storage::DynNodeStoragePtr node_storage
                     auto value = from_string(lexical_backend.lexical_form);
 
                     return node_storage.find_id(storage::view::ValueLiteralBackendView{.datatype = dt_id,
-                                                                                             .value = std::move(value)});
+                                                                                         .value = std::move(value)});
                 }
 
                 // Default case.
                 // This node storage doesn't have specialized storage for the value and the new one also doesn't
-                auto const dtype_iri_view = node_storage.find_iri_backend(lexical_backend.datatype_id);
+                auto const dtype_iri_view = handle_.storage().find_iri_backend(lexical_backend.datatype_id);
```

Comment:
Good catch. Took me a bit though, to understand why this is correct. 

It would make the code easier to read if there was a something like 
```
auto& source_node_storage = handle_.storage();
```